nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/security/PermissionAutomationTest.java

```diff
@@ -78,6 +82,15 @@
     @Inject
     AutomationService automationService;
 
+    @Inject
+    private UserManager userManager;
+
+    /**
+     * Transient user for test. It is created before each test and deleted after each test. Do NOT manipulate this
+     * object during the test.
+     */
+    private DocumentModel _user;
```

Comment:
Just call it `testUser`, no need for an initial `_`. And actually in `@After` you only need the user id so just store that, `testUserId`, not the whole object.